BridgeCareApp/VuejsApp/src/components/Criteria.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+    <v-container fluid grid-list-xl>
+        <v-layout row wrap>
+            <v-flex xs12>
+                <v-btn v-on:click="onLaunchCriteriaEditor()">Launch Criteria Editor</v-btn>
+            </v-flex>
+            <v-flex xs12>
+                <div>Before Parsing: {{beforeParsing}}</div>
+            </v-flex>
+            <v-flex>
+                <div v-show="afterParsing !== ''">After Parsing: {{afterParsing}}</div>
+            </v-flex>
+            <v-flex xs12>
+                <CriteriaEditor :showCriteriaEditor="showCriteriaEditor" @applyCriteria="onApplyCriteria"/>
+            </v-flex>
+        </v-layout>
+    </v-container>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import {Component} from "vue-property-decorator";
+    import CriteriaEditor from "../shared/CriteriaEditor.vue";
+
+    @Component({
+        components: {CriteriaEditor}
+    })
+    export default class Criteria extends Vue {
+        beforeParsing: string = "[ADTTOTAL]='50' OR [ADTYEAR]='1999' AND [AGE]>'30' OR [APPRALIGN]>='7' OR [APPRALIGN]<='5'";
```

Comment:
Is this hard-coded for a reason?

Reply:
This is hardcoded because currently there is no view that is using the CriteriaEditor. Even if there was, there currently isn't any web services powering those views with criteria to use. I did this so that we could display working functionality to the client during sprint reviews. It'll be gone once things start coming online for the app and we have components that actually use this.